graphql/schema/bug.graphql

```diff
@@ -36,6 +36,8 @@ type Bug {
   title: String!
   labels: [Label!]!
   author: Identity!
+  actors: [Identity]!
```

Comment:
After talking with @sandhose, we'll go for the connection here. But we can merge like this and I'll add it. It's a bit tricky.

Reply:
Alright ! Ping me when it's ready. I'd like to see how that's implemented 👍